Index: ../public/src/es5/scenes/scene from class.js
===================================================================
--- ../public/src/es5/scenes/scene from class.js
+++ ../public/src/es5/scenes/scene from class.js
@@ -26,8 +26,8 @@
     width: 800,
     height: 600,
     backgroundColor: '#000000',
     parent: 'phaser-example',
-    scene: MyScene
+    scene: [ MyScene ]
 };
 
 var game = new Phaser.Game(config);
