Index: ../public/src/es5/scenes/scene injection map utf8.js
===================================================================
--- ../public/src/es5/scenes/scene injection map utf8.js
+++ ../public/src/es5/scenes/scene injection map utf8.js
@@ -32,8 +32,8 @@
     type: Phaser.AUTO,
     width: 800,
     height: 600,
     parent: 'phaser-example',
-    scene: MyScene
+    scene: [ MyScene ]
 };
 
 var game = new Phaser.Game(config);
