Index: ../public/src/es5/textures/edit texture hue shift.js
===================================================================
--- ../public/src/es5/textures/edit texture hue shift.js
+++ ../public/src/es5/textures/edit texture hue shift.js
@@ -10,18 +10,18 @@
 };
 
 var originalTexture;
 var newTexture;
-var context; 
+var context;
 
 var game = new Phaser.Game(config);
 
-function preload() 
+function preload()
 {
     this.load.image('dude', 'assets/sprites/phaser-dude.png');
 }
 
-function create() 
+function create()
 {
     originalTexture = this.textures.get('dude').getSourceImage();
 
     newTexture = this.textures.createCanvas('dudeNew', originalTexture.width, originalTexture.height);
@@ -39,9 +39,9 @@
 function hueShift ()
 {
     var pixels = context.getImageData(0, 0, originalTexture.width, originalTexture.height);
 
-    for (i = 0; i < pixels.data.length / 4; i++)
+    for (var i = 0; i < pixels.data.length / 4; i++)
     {
         processPixel(pixels.data, i * 4, 0.1);
     }
 
