Index: ../public/src/es5/physics/matterjs/particles follow body.js
===================================================================
--- ../public/src/es5/physics/matterjs/particles follow body.js
+++ ../public/src/es5/physics/matterjs/particles follow body.js
@@ -38,24 +38,16 @@
 
     var emitter = particles.createEmitter({
         frame: 'blue',
         speed: {
-            onEmit: function (particle, key, t, value)
-            {
-                return ship.body.speed;
-            }
+            onEmit: (particle, key, t, value) => ship.body.speed
         },
         lifespan: {
-            onEmit: function (particle, key, t, value)
-            {
-                return Phaser.Math.Percent(ship.body.speed, 0, 300) * 20000;
-            }
+            onEmit: (particle, key, t, value) => Phaser.Math.Percent(ship.body.speed, 0, 300) * 20000
         },
         alpha: {
-            onEmit: function (particle, key, t, value)
-            {
-                return Phaser.Math.Percent(ship.body.speed, 0, 300) * 1000;
-            }
+            onEmit: (particle, key, t, value) => Phaser.Math.Percent(ship.body.speed, 0, 300) * 1000
+
         },
         scale: { start: 1.0, end: 0 },
         blendMode: 'ADD'
     });
