Index: ../public/src/es5/audio/HTML5 Audio/markers pause resume.new
===================================================================
--- ../public/src/es5/audio/HTML5 Audio/markers pause resume.new
+++ ../public/src/es5/audio/HTML5 Audio/markers pause resume.new
@@ -9,9 +9,9 @@
     height: 600,
     scene: {
         preload: preload,
         create: create,
-        update: updatePauseResumeButton
+        update: update
     },
     pixelArt: true,
     audio: {
         disableWebAudio: true
@@ -124,9 +124,9 @@

     pauseResumeButtonText = this.add.bitmapText(640, 387, 'nokia', '', 16);
 }

-function updatePauseResumeButton()
+function update()
 {
     if(fx.isPaused)
     {
         pauseResumeButtonText.text = 'resume';
