Index: ../public/src/es5/game objects/render texture/dynamic tilemap to render texture.js
===================================================================
--- ../public/src/es5/game objects/render texture/dynamic tilemap to render texture.js
+++ ../public/src/es5/game objects/render texture/dynamic tilemap to render texture.js
@@ -22,9 +22,9 @@
 }
 
 function create ()
 {
-    map = this.make.tilemap({ key: 'map' });
+    var map = this.make.tilemap({ key: 'map' });
 
     var tiles = map.addTilesetImage('Desert', 'tiles');
 
     layer = map.createLayer('Ground', tiles, 0, 0).setVisible(false);
