Index: ../public/src/es5/game objects/render texture/graphics to render texture.js
===================================================================
--- ../public/src/es5/game objects/render texture/graphics to render texture.js
+++ ../public/src/es5/game objects/render texture/graphics to render texture.js
@@ -43,9 +43,9 @@
     graphics.fillStyle(color, 1.0);
     graphics.beginPath();
     graphics.moveTo(cx, cy - outerRadius);
 
-    for (i = 0; i < spikes; i++)
+    for (var i = 0; i < spikes; i++)
     {
         x = cx + Math.cos(rot) * outerRadius;
         y = cy + Math.sin(rot) * outerRadius;
         graphics.lineTo(x, y);
