Index: ../public/src/es5/animation/create from sprite config.new
===================================================================
--- ../public/src/es5/animation/create from sprite config.new
+++ ../public/src/es5/animation/create from sprite config.new
@@ -24,9 +24,8 @@
     this.anims.create({ key: 'ruby', frames: this.anims.generateFrameNames('gems', { prefix: 'ruby_', end: 6, zeroPad: 4 }), repeat: -1 });
     this.anims.create({ key: 'square', frames: this.anims.generateFrameNames('gems', { prefix: 'square_', end: 14, zeroPad: 4 }), repeat: -1 });

     //  The Sprite config
-
     const config = {
         key: 'gems',
         x: { randInt: [ 0, 800 ] },
         y: { randInt: [ 0, 300 ] },
@@ -34,9 +33,9 @@
         anims: 'ruby'
     };

     //  Make 16 sprites using the config above
-    for (var i = 0; i < 16; i++)
+    for (let i = 0; i < 16; i++)
     {
         this.make.sprite(config);
     }

