Index: ../public/src/es5/physics/arcade/body controls.new
===================================================================
--- ../public/src/es5/physics/arcade/body controls.new
+++ ../public/src/es5/physics/arcade/body controls.new
@@ -17,9 +17,9 @@
 };

 var sprite;

-new Phaser.Game(config);
+var game = new Phaser.Game(config);

 function preload ()
 {
     this.load.image('mushroom', 'assets/sprites/mushroom2.png');
