Index: ../public/src/es5/physics/arcade/custom separate.new
===================================================================
--- ../public/src/es5/physics/arcade/custom separate.new
+++ ../public/src/es5/physics/arcade/custom separate.new
@@ -16,9 +16,9 @@
         create: create
     }
 };

-new Phaser.Game(config);
+var game = new Phaser.Game(config);

 function preload ()
 {
     this.load.image('block', 'assets/sprites/block.png');
