Index: ../public/src/es5/physics/arcade/draggable body.new
===================================================================
--- ../public/src/es5/physics/arcade/draggable body.new
+++ ../public/src/es5/physics/arcade/draggable body.new
@@ -17,9 +17,9 @@
 };

 var block;

-new Phaser.Game(config);
+var game = new Phaser.Game(config);

 function preload ()
 {
     this.load.image('block', 'assets/sprites/block.png');
