Index: ../public/src/es5/physics/arcade/gravity.new
===================================================================
--- ../public/src/es5/physics/arcade/gravity.new
+++ ../public/src/es5/physics/arcade/gravity.new
@@ -15,9 +15,9 @@
         create: create
     }
 };

-new Phaser.Game(config);
+var game = new Phaser.Game(config);

 function preload ()
 {
     this.load.image('block', 'assets/sprites/block.png');
