Index: ../public/src/es5/physics/arcade/group vs self.new
===================================================================
--- ../public/src/es5/physics/arcade/group vs self.new
+++ ../public/src/es5/physics/arcade/group vs self.new
@@ -15,9 +15,9 @@
         create: create
     }
 };

-new Phaser.Game(config);
+var game = new Phaser.Game(config);

 function preload()
 {
     this.load.image('ball', 'assets/sprites/shinyball.png');
