Index: ../public/src/es5/physics/arcade/overlap zone.new
===================================================================
--- ../public/src/es5/physics/arcade/overlap zone.new
+++ ../public/src/es5/physics/arcade/overlap zone.new
@@ -20,9 +20,9 @@
 };

 var zone;

-new Phaser.Game(config);
+var game = new Phaser.Game(config);

 function preload ()
 {
     this.load.image('block', 'assets/sprites/block.png');
