Index: ../public/src/es5/physics/arcade/world controls.js
===================================================================
--- ../public/src/es5/physics/arcade/world controls.js
+++ ../public/src/es5/physics/arcade/world controls.js
@@ -52,9 +52,9 @@
 var balls;
 var blocks;
 var graphics;
 
-new Phaser.Game(config);
+var game = new Phaser.Game(config);
 
 function preload ()
 {
     this.load.image('ball', 'assets/sprites/yellow_ball.png');
