Index: ../public/src/es5/geom/line/copy from.js
===================================================================
--- ../public/src/es5/geom/line/copy from.js
+++ ../public/src/es5/geom/line/copy from.js
@@ -11,15 +11,15 @@
 var game = new Phaser.Game(config);
 
 function create ()
 {
-    graphics = this.add.graphics({ lineStyle: { width: 3, color: 0xaa00aa } });
+    var graphics = this.add.graphics({ lineStyle: { width: 3, color: 0xaa00aa } });
 
-    pointerLine = new Phaser.Geom.Line(0, 0, 0, 200);
+    var pointerLine = new Phaser.Geom.Line(0, 0, 0, 200);
 
-    lines = [];
+    var lines = [];
 
-    for(var i = 0; i < 60; i++)
+    for(var k = 0; k < 60; k++)
     {
         lines.push(new Phaser.Geom.Line(0, 0, 0, 0));
     }
 
