Index: ../public/src/es5/game objects/graphics/generate texture to tilesprite.new
===================================================================
--- ../public/src/es5/game objects/graphics/generate texture to tilesprite.new
+++ ../public/src/es5/game objects/graphics/generate texture to tilesprite.new
@@ -41,9 +41,9 @@
     graphics.fillStyle(color, 1.0);
     graphics.beginPath();
     graphics.moveTo(cx, cy - outerRadius);

-    for (i = 0; i < spikes; i++) {
+    for (var i = 0; i < spikes; i++) {
         x = cx + Math.cos(rot) * outerRadius;
         y = cy + Math.sin(rot) * outerRadius;
         graphics.lineTo(x, y);
         rot += step;
