Index: ../public/src/es5/game objects/graphics/phaser logo shift.js
===================================================================
--- ../public/src/es5/game objects/graphics/phaser logo shift.js
+++ ../public/src/es5/game objects/graphics/phaser logo shift.js
@@ -26,9 +26,9 @@
     var hsv = Phaser.Display.Color.HSVColorWheel();
 
     colors = [];
 
-    for (var i = 0; i < hsv.length; i += 4)
+    for (let i = 0; i < hsv.length; i += 4)
     {
         colors.push(hsv[i].color);
     }
 
@@ -42,9 +42,9 @@
         thickness: 10,
         alpha: 1
     };
 
-    for (var i = 0; i < logos; i++)
+    for (let i = 0; i < logos; i++)
     {
         s.push(0);
     }
 
