Index: ../public/src/es5/game objects/particle emitter/particle processor.js
===================================================================
--- ../public/src/es5/game objects/particle emitter/particle processor.js
+++ ../public/src/es5/game objects/particle emitter/particle processor.js
@@ -11,9 +11,9 @@
 };
 
 var SECOND = 1000;
 
-new Phaser.Game(config);
+var game = new Phaser.Game(config);
 
 function preload ()
 {
     this.load.atlas('flares', 'assets/particles/flares.png', 'assets/particles/flares.json');
