Index: ../public/src/es5/geom/triangle/copy from.js
===================================================================
--- ../public/src/es5/geom/triangle/copy from.js
+++ ../public/src/es5/geom/triangle/copy from.js
@@ -11,15 +11,15 @@
 var game = new Phaser.Game(config);
 
 function create ()
 {
-    graphics = this.add.graphics({ lineStyle: { width: 3, color: 0xaaaa00 } });
+    var graphics = this.add.graphics({ lineStyle: { width: 3, color: 0xaaaa00 } });
 
-    pointerTriangle = new Phaser.Geom.Triangle.BuildEquilateral(100, 100, 150);
+    var pointerTriangle = new Phaser.Geom.Triangle.BuildEquilateral(100, 100, 150);
 
-    triangles = [];
+    var triangles = [];
 
-    for(var i = 0; i < 45; i++)
+    for(var k = 0; k < 45; k++)
     {
         triangles.push(new Phaser.Geom.Triangle());
     }
 
